### PR TITLE
ref(replays): adjust widget tooltips and headers

### LINKED ISSUE #54284: [Replay List] Adjust dead & rage click table headers
Some feedback we got is:

- Move the tooltip from the right column, to the left
- Drop the label on the right column

Image

### PR DESCRIPTION
**Pull request by michellewzhang**
Fixes https://github.com/getsentry/sentry/issues/54284

SCR-20230809-kvmy

### CODE REVIEW

File: static/app/views/replays/replayTable/types.tsx [hunk 1]
```diff
@@ -11,4 +11,6 @@ export enum ReplayColumn {
   OS = 'os',
   REPLAY = 'replay',
   SLOWEST_TRANSACTION = 'slowestTransaction',
+  COUNT_DEAD_CLICKS_NO_HEADER = 'countDeadClicksNoHeader',
+  COUNT_RAGE_CLICKS_NO_HEADER = 'countRageClicksNoHeader',
```
Comment (ryan953) [inline static/app/views/replays/replayTable/types.tsx:15]: Nit: can we sort this enum? i find it easier to scan and find things that way.

**Review by ryan953 — APPROVED** [review]
Code looks good, but lets wait for jasmin to take a look at the screen shots.

There's less stuff going on in the header, but for me the count number itself is a bit lost now and harder to spot when i'm looking at the rows.

Should we put skill/sad-face emoji's in there (similar to how we have the fire icon for errors?)

**Review by jas-kas — APPROVED** [review]

**Comment by jas-kas** [comment]
> Should we put skill/sad-face emoji's in there (similar to how we have the fire icon for errors?)

Sure! Not opposed to this. Would this be colour-coded in any way? The fire icon is always red for errors.

**Comment by michellewzhang** [comment]
> > Should we put skill/sad-face emoji's in there (similar to how we have the fire icon for errors?)
> 
> Sure! Not opposed to this. Would this be colour-coded in any way? The fire icon is always red for errors.

cc @Jesse-Box for his thoughts on icons & colors when he's back in office!

**Comment by michellewzhang** [comment]
Here's what it might look like with icons:

SCR-20230808-jdjd

**Comment by michellewzhang** [comment]
moving forward without icons for now since Jesse is OOO- I'll sync up with him directly about the icons

**Review by ryan953 — APPROVED** [review]

**Pull request merged**